cli/tests/test_help_command.rs

```diff
@@ -84,3 +79,145 @@ fn test_help() {
     For more information, try '--help'.
     "#);
 }
+
+#[test]
+fn test_help_category() {
+    let test_env = TestEnvironment::default();
+    // Now that we have a custom help output, capture it
+    let help_cmd_stdout = test_env.jj_cmd_success(test_env.env_root(), &["help"]);
+    insta::assert_snapshot!(help_cmd_stdout, @r#"
+    Jujutsu (An experimental VCS)
```

Comment:
I think it's a bad idea to hard code the help output into the test.

Reply:
It's not so bad because the snapshot can be updated relatively easily, but I would add some regex pattern to omit commands and options lists.
https://insta.rs/docs/filters/